client/src/components/Footer.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import { urlFor } from "../util";
+
+const FooterLogo = styled.img.attrs({
+  src: urlFor("footerLogo.png"),
+})`
+  height: 2.5em;
+  padding: 1.25em;
+  box-sizing: initial;
+`;
+
```

Comment:
remove these 2 extra spaces, leave only one space between component definitions

Reply:
Can run prettier to fix

Reply:
Resolved